openassessment/xblock/test/test_openassessment.py

```diff
@@ -156,6 +157,10 @@ def test__create_ui_models__no_leaderboard_if_teams_enabled(self, xblock):
         self.assertEqual(models[1], UI_MODELS["grade"])
 
     @scenario('data/basic_scenario.xml')
+    @patch(
```

Comment:
Is this necessary?

Reply:
`openedx` code suppose to get run and it is not define in a normal environment. There isn't safety around it either. The test would through error if I do not mock it.

Reply:
I see I already asked this question several days ago xD